GVFS/GVFS.Platform.Mac/MacDaemonManager.cs

```diff
@@ -0,0 +1,79 @@
+using GVFS.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.Platform.Mac
+{
+    /// <summary>
+    /// Class to query the configured services on macOS
+    /// </summary>
+    public class MacDaemonManager
+    {
+        private const string LaunchCtlPath = @"/bin/launchctl";
+        private const string LaunchCtlArg = @"list";
+
+        private IProcessHelper processHelper;
+
+        public MacDaemonManager()
+        : this(new ProcessHelperImpl())

```

Comment:
Can we get by with deleting this constructor and have the caller create the dependency and pass it in?